src/components/Scoreboard/Score.mock.js

```diff
@@ -0,0 +1,20 @@
+export default {
+    players: [
+        {
+            participantId: 'qq-ewq-sdas',
+            displayName: 'Natalie Portman',
+            aggregateScore: 340,
+            correct: 'correct',
+            code: '.select-all',
+            time: 55
+        },
+        {
+            participantId: 'dsda-ffd-12ad',
+            displayName: 'Keira Knightley',
+            aggregateScore: 550,
+            correct: 'partial',
+            code: '.select-not-all',
+            time: 90
+        }
+    ]
+}
```

Comment:
is it possible to distinguish between Natalie Portman and Keira Knightley yet? 😆

Reply:
No need to distinguish, they are good together